lib/multivm/src/lib.rs

```diff
@@ -22,6 +22,40 @@ pub enum ExecutionVersion {
     V3 = 3,
 }
 
+impl ExecutionVersion {
+    // NOTE: V1 and V2 have a slight chance of being off as they've been backfilled.
+    // If you find a divergence in what you expect and the actual value, most likely a bug.
+
+    /// verification key hash generated from zksync-os v0.0.21, zksync-airbender v0.4.4 and zkos-wrapper v0.4.3
+    const V1_VK_HASH: &'static str =
```

Comment:
I actually find this suboptimal and much rather prefer the typed version (so interfaces don't depend on &'static str that may be anything). That said, can be done as a follow-up. LMK if you have any thoughts on it.